autogpt/llm/llm_utils.py

```diff
@@ -71,7 +71,7 @@ def _wrapped(*args, **kwargs):
 
 
 def call_ai_function(
-    function: str, args: list, description: str, model: str | None = None
+    function: str, args: list, description: str, model: str | None = None, config=None
```

Comment:
Untyped argument

Reply:
same here let's extend my py to autogpt/commands so we slowly get everything fully typed

Reply:
theres 30+ changes required to extend mypy here. Thats more than reasonable to include in this PR as it will drown out the actual changes of the PR